src/utils/global/tooltip/tooltip.module.scss

```
@@ -0,0 +1,39 @@
+@import '../../../variables.scss';
+
+.sharedStyles {
+  background-color: $tooltip-background-color;
+  border-radius: 0.5rem;
+  color: white;
+  font-size: 1rem;
+  left: calc(100% + .75rem);
+  padding: 0.5rem;
+  position: absolute;
+  transition: $transition;
+
+}
+
+.triangle {
+  position: absolute;
+  margin-top: -5px;
+  right: 100%;
+  top: 50%;
+  border-width: 5px;
+  border-style: solid;
+  border-color: transparent $tooltip-background-color transparent transparent;
+}
+
+.children {
+  position: relative;
+  display: flex;
+  height: 100%;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+}
+.wrapper {
```

```suggestion

.wrapper {
```
Nitpick, linebreak between blocks